openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Enable users to switch between viewing and writing CourseTalk reviews.
+ */
+
+export class CourseTalkReviews {  // eslint-disable-line import/prefer-default-export
+  constructor() {
+    const $courseTalkToggleReadWriteReviews = $('.toggle-read-write-reviews');
+    const $courseTalkScriptObject = $('.coursetalk-reviews-script');
+
+    const readSrc = '//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js';
```

Comment:
Would it be possible to have the configuration live outside the code? It is going to be hard to track this down if this ever needs to change. Maybe this should be configured in Django and passed down to the client.